justfile

```diff
@@ -0,0 +1,14 @@
+default:
+    @just --list
+
+# Auto-format the source tree
+fmt:
+    treefmt
```

Comment:
what is treefmt?

Reply:
A command line tool to rule all formatting, [treefmt](https://github.com/numtide/treefmt)

Just a preference. We should stick to Rust/Python ecosystem tools, better accepted for the community.